core/matchers/src/main/kotlin/au/com/dius/pact/core/matchers/util/CollectionUtils.kt

```diff
@@ -29,4 +29,4 @@ fun <E> List<E>.padTo(size: Int, item: E): List<E> {
     }
     return list
   }
-}
+}
```

Comment:
```suggestion
}

```

New line existed in original version, https://github.com/mitre/pact-jvm/blob/3bc07b834f651f1aa17663416786cdae7a9ce0d6/core/matchers/src/main/kotlin/au/com/dius/pact/core/matchers/util/CollectionUtils.kt.